app/controllers/restaurants_controller.rb

```diff
@@ -4,21 +4,25 @@ class RestaurantsController < ApplicationController
   def index
     restaurants = Restaurant.all
 
-    render json: { restaurants: restaurants.as_json(include: :reviews) }
+    render json: { restaurants: restaurants.as_json(include: { holidays: {}, reviews: {} }) }
```

Comment:
```suggestion
    render json: { restaurants: restaurants.as_json(include: [:holidays, :reviews]) }
```
でいける気がする（includeは柔軟によしなにしてくれた記憶）